Expecto/Expecto.fs

```diff
@@ -1613,6 +1626,83 @@ module Tests =
   /// The default configuration for Expecto.
   let defaultConfig = ExpectoConfig.defaultConfig
 
+  module Args =
+    open Microsoft.FSharp.Core
+
+    type Parser<'a> = (string[] * int * int) -> Result<'a,string> * int
+
+    let parseOptions (options:(string * string * Parser<_>) list) (strings:string[]) =
+      let rec updateUnknown unknown last length =
+        if length=0 then unknown
+        else updateUnknown (strings.[last]::unknown) (last-1) (length-1)
+      let rec collect help unknown args paramCount i =
+        if i>=0 then
+          let si = strings.[i]
```

Comment:
`sCurr` or something easier to understand?

Reply:
currentArg